app/assets/stylesheets/layouts/dark.scss

```diff
@@ -43,6 +43,12 @@ $font_color_flash_breadcrumbs: #DDDDDD;
 .flash.error { background-color: $bg_color_flash_error; }
 .flash.inbox { background-color: $bg_color_flash_inbox; color: $font_color_flash_inbox; }
 
+.flash.success {
+  background-color: #5B655B;
+  color: #bbb;
+}
+//TODO: This probably overrides previous .flash.success?
```

Comment:
This is gonna mess up the background color of the above `.flash.success`, because before it was `.success` and so had lower precedence; can we remove `background-color` from this rule? `color: …` can stay.

Reply:
aha! is that was what going on. yeah I'll remove that and merge the font color in